arboretum/azure_cloud/fetchers/fetch_sub_assessments.py

```diff
@@ -0,0 +1,57 @@
+"""Azure Cloud sub assessment fetcher API.
```

Comment:
You will have to add a license preamble here. You can take: https://github.com/ComplianceAsCode/auditree-arboretum/pull/66/files#diff-a09c14cd37bde12891584520a7a24d627b43c73a5b508205afaa3b03ed6f5f72R3